app/src/main/java/org/systers/mentorship/view/activities/LoginActivity.kt

```diff
@@ -81,4 +81,8 @@ class LoginActivity : BaseActivity() {
         loginViewModel.successful.removeObservers(this)
         loginViewModel.successful.value = null
     }
+    override fun onBackPressed() {
+        super.onBackPressed()
+        finishAffinity()
```

Comment:
Instead of doing `finishActivity()` here and in the SignUpActivity.kt, couldn't you do that on the [logout "method" on SettingsActivity.kt](https://github.com/systers/mentorship-android/blob/develop/app/src/main/java/org/systers/mentorship/view/activities/SettingsActivity.kt#L22) ? Maybe replacing `finish()` with `finishAffinity()`? cc @m-murad